src/pe_reports/helpers/bulletin/bulletin_generator.py

```diff
@@ -0,0 +1,274 @@
+
+    cont = input("Would you like to add more content (Y/N): ")
+    if cont == "Y":
+        text = html_builder(text)
+
+    return text
+
+
+bulletin_path = os.path.dirname(os.path.realpath(__file__))
+
+
+def generate_cybersix_bulletin(
+    id,
+    user_text="",
+    output_directory="/var/www/cybersix_bulletins",
+    filename="_Bulletin.pdf",
+):
+    """Generate a bulletin based on a provided cybersix id."""
+    if not filename.endswith(".pdf"):
+        filename = filename + ".pdf"
+
+    templateLoader = jinja2.FileSystemLoader(searchpath=bulletin_path)
+    templateEnv = jinja2.Environment(loader=templateLoader, autoescape=True)
+    templateEnv.filters["date_format"] = date_format
+    TEMPLATE_FILE = "bulletin_template.html"
+    template = templateEnv.get_template(TEMPLATE_FILE)
+    resp = get_post(id)
+
+    for post in resp["intel_items"]:
+
+        outputText = template.render(
+            post,
+            user_provided_content=user_text,
+            Stakeholder_Name="Posture and Exposure",
+        )
+        html_file = open(bulletin_path + "/bulletin_template_filled.html", "w")
+        html_file.write(outputText)
+        html_file.close()
+
+        options = {
+            "page-size": "Letter",
+            "margin-top": "1.3in",
+            "margin-right": "0in",
+            "margin-bottom": "1.1in",
+            "margin-left": "0in",
+            "dpi": 96,
+            "encoding": "UTF-8",
+            "custom-header": [("Accept-Encoding", "gzip")],
+            "cookie": [
+                ("cookie-empty-value", '""'),
+                ("cookie-name1", "cookie-value1"),
+                ("cookie-name2", "cookie-value2"),
+            ],
+            "no-outline": None,
+            "header-html": bulletin_path + "/header.html",
+            "header-spacing": -5,
+            "footer-right": "[page] of [topage]       \t\t\t\t",
+            "footer-left": "       \t\t\tPosture & Exposure",
+            "footer-spacing": 8,
+            "footer-html": bulletin_path + "/footer.html",
+            "enable-local-file-access": True,
+            "disable-smart-shrinking": True,
+        }
+        out_path = output_directory + "/" + filename
+
+        pdfkit.from_file(
+            [bulletin_path + "/bulletin_template_filled.html"],
+            out_path,
+            options=options,
+            verbose=True,
+        )
+
+
+def generate_creds_bulletin(
+    breach,
+    org_name,
+    user_text,
+    output_directory="/var/www/cred_bulletins",
+    filename="_Bulletin.pdf",
+):
+    """Generate a credential breach bulletin."""
+    LOGGER.info("generating creds bulletin")
+    templateLoader = jinja2.FileSystemLoader(searchpath=bulletin_path)
+    templateEnv = jinja2.Environment(loader=templateLoader, autoescape=True)
+    templateEnv.filters["date_format"] = date_format
+    TEMPLATE_FILE = "creds_bulletin_template.html"
+    template = templateEnv.get_template(TEMPLATE_FILE)
```

Comment:
More camel case that should be snake case:
```suggestion
    template_loader = jinja2.FileSystemLoader(searchpath=bulletin_path)
    template_env = jinja2.Environment(loader=template_loader, autoescape=True)
    template_env.filters["date_format"] = date_format
    TEMPLATE_FILE = "creds_bulletin_template.html"
    template = template_env.get_template(TEMPLATE_FILE)
```